pkg/controller/common/util.go

```diff
@@ -1088,3 +1140,18 @@ func NewImportDataVolume(name string) *cdiv1.DataVolume {
 		},
 	}
 }
+
+// GetCloneSourceNameAndNamespace returns the name and namespace of the cloning source
```

Comment:
maybe return NamespacedName?

Reply:
I prefer to keep this one if that's fine